receiver/jaegerreceiver/testdata/bad_empty_config.yaml

```diff
@@ -1,15 +0,0 @@
-receivers:
```

Comment:
Can you confirm the removal of those files are intended?

Reply:
yes, I merged them into https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/14476/files/1db4f14070ab8b86c65ddb8d8d161fe68d827a15#diff-56f282193f5228abfb85e5e670e12b201672b82d126baa296a01268e65f6954cR54